docs/source/guide/predictions.md

```diff
@@ -640,6 +655,326 @@ from label_studio_converter import brush
 
+## Import OCR pre-annotations 
+
+Import pre-annotations for optical character recognition (OCR), such as output from [tesseract like in this example blog post](/blog/Improve-OCR-quality-with-Tesseract-and-Label-Studio.html). 
+
+In this example, import pre-annotations for OCR tasks using the [OCR template](/templates/optical_character_recognition.html):
+
+```xml
+<View>
+  <Image name="image" value="$image"/>
+  <Labels name="label" toName="image">
+    <Label value="Text" background="green"/>
+    <Label value="Handwriting" background="blue"/>
+  </Labels>
+  <Rectangle name="bbox" toName="image" strokeWidth="3"/>
+  <Polygon name="poly" toName="image" strokeWidth="3"/>
+  <TextArea name="transcription" toName="image" editable="true" perRegion="true" required="true" maxSubmissions="1" rows="5" placeholder="Recognized Text" displayMode="region-list"/>
+</View>
+```
+
+### Example JSON
+
+This example JSON contains one task with three results dictionaries, one for each type of tag in the labeling configuration: Rectangle, Labels, and TextArea. 
+
+<br/>
+{% details <b>Click to expand the example image JSON</b> %}
+Save this example JSON as a file to import it into Label Studio, for example, `example_prediction_task.json`.
+
+{% codeblock lang:json %}
+{
+   "data":{
+      "ocr":"/data/upload/receipt_00523.png"
+   },
+   "predictions":[
+      {
+         "model_version":"best_ocr_model_1_final",
+         "result":[
+            {
+               "original_width":864,
+               "original_height":1296,
+               "image_rotation":0,
+               "value":{
+                  "x":16.203703703703702,
+                  "y":66.74382716049382,
+                  "width":68.28703703703704,
+                  "height":5.015432098765432,
+                  "rotation":0
+               },
+               "id":"bb5",
+               "from_name":"bbox",
+               "to_name":"image",
+               "type":"rectangle"
+            },
+            {
+               "original_width":864,
+               "original_height":1296,
+               "image_rotation":0,
+               "value":{
+                  "x":16.203703703703702,
+                  "y":66.74382716049382,
+                  "width":68.28703703703704,
+                  "height":5.015432098765432,
+                  "rotation":0,
+                  "text":[
+                     "CASH 100.000 \nCHANGED 90.000"
+                  ]
+               },
+               "id":"bb5",
+               "from_name":"transcription",
+               "to_name":"image",
+               "type":"textarea",
+               "score":0.89
+            },
+            {
+               "original_width":864,
+               "original_height":1296,
+               "image_rotation":0,
+               "value":{
```

Comment:
please, use spaces for json, e.g. "original_width":864 => "original_width": 864. There must be autoconverters, e.g. pycharm or online tools.

Reply:
got it, I'll check what my pycharm can do for me :)